druid/examples/identity.rs

```diff
@@ -16,8 +16,9 @@
 //!
 //! This example is fairly contrived; the basic idea is that there are three
 //! rows of widgets, each containing a ColorWell and two buttons. One button
-//! 'freezes' the ColorWell, sending it a color to display. The second button
-//! 'unfreezes' the ColorWell, which makes it start displaying random colors.
+//! 'pins' the ColorWell, sending it a color to display. The second button
+//! 'syncs' the ColorWell, which makes it start displaying random colors. just
+//! like the top widget
```

Comment:
I found this confusing on the first read-through because the "top widget" hasn't been mentioned yet.

Reply:
oh yeah that doesnt really make sense, thanks for letting me know.

Reply:
let me know if it makes more sense now.